src/Sylius/Behat/Service/Setter/CookieSetter.php

```diff
@@ -83,7 +83,7 @@ private function shouldMinkSessionBePrepared(Session $session): bool
             return true;
         }
 
-        if ($driver instanceof ChromeDriver && $driver->isStarted() === false) {
```

Comment:
What it means: every time administrator is logging in while testing JS scenarios - refresh the browser.